libtls: Add implementation for enums

bytinbit opened the pull request:
Squashed commits:
libtls: Enforce consistent code-style
libtls: Add alert types
libtls: Add handshake types and extensions
libtls: Add handshake type names, error persists
libtls: Add TLS version names

Commit:
add tls 1.3 constants

bytinbit committed:
Merge pull request #2 from bytinbit/feature-constants

TLS 1.3 constants implemented

ryru commented on src/libtls/tls.c:
```diff
@@ -40,30 +42,53 @@ ENUM(tls_content_type_names, TLS_CHANGE_CIPHER_SPEC, TLS_APPLICATION_DATA,
+	"HelloRequest");
+ENUM_NEXT(tls_handshake_type_names,
+		TLS_CLIENT_HELLO, TLS_HELLO_RETRY_REQUEST, TLS_HELLO_REQUEST,
+		// first                 last                      prev
```
please remove comment

Commit:
libtls: Add new cipher suites
libtls: Enforce consistent code-style
libtls: Add alert types
libtls: Add handshake types and extensions
libtls: Add handshake type names, error persists
libtls: Add TLS version names

bytinbit commented:
Closes #4
